eppoclient/client.go

```diff
@@ -80,6 +81,76 @@ func (ec *EppoClient) GetJSONAssignment(flagKey string, subjectKey string, subje
 	return variation, err
 }
 
+type BanditResult struct {
+	Variation string
+	Action    *string
+}
+
+func (ec *EppoClient) GetBanditAction(flagKey string, subjectKey string, subjectAttributes ContextAttributes, actions map[string]ContextAttributes, defaultVariation string) BanditResult {
+	// ignoring the error here as we can always proceed with default variation
+	variation, _ := ec.GetStringAssignment(flagKey, subjectKey, subjectAttributes.toGenericAttributes(), defaultVariation)
+
+	bandit, err := ec.configurationStore.getBanditConfiguration(variation)
+	if err != nil {
+		// no bandit configuration
+		return BanditResult{
+			Variation: variation,
+			Action:    nil,
+		}
+	}
+
+	evaluation := bandit.ModelData.evaluate(banditEvaluationContext{
+		flagKey:           flagKey,
+		subjectKey:        subjectKey,
+		subjectAttributes: subjectAttributes,
+		actions:           actions,
+	})
+	if evaluation.result == nil {
```

Comment:
This seems like an error situation--at this point should we instead return the default value?

Reply:
`BanditResult{variation, nil}` is the default value in this case. This function does not receive a default action to return (only default variation).

This is the same interface/behavior as we have in Python. cc @schmit